examples/shader/post_process_pass.rs

```diff
@@ -0,0 +1,383 @@
+        let Ok(view_target) = self.query.get_manual(world, view_entity) else {
+            return Ok(());
+        };
+
+        // Get the pipeline resource that contains the global data we need to create the render pipeline
+        let post_process_pipeline = world.resource::<PostProcessPipeline>();
+        // The pipeline cache is a cache of all previously created pipelines.
+        // It's required to avoid creating a new pipeline each frame.
+        let pipeline_cache = world.resource::<PipelineCache>();
+
+        // Get the pipeline data for the current view
+        let Some(pipeline) = pipeline_cache.get_render_pipeline(post_process_pipeline.pipeline_id) else {
+            return Ok(());
+        };
+
+        // Get the settings uniform binding
+        let settings_uniforms = world.resource::<ComponentUniforms<PostProcessSettings>>();
+        let Some(settings_binding) = settings_uniforms.uniforms().binding() else {
+            return Ok(());
+        };
+
+        // Get the TextureView used for post processing effects in bevy
+        let post_process = view_target.post_process_write();
+
+        // The bind_group gets created each frame.
+        // It's important for this to match the BindGroupLayout defined in the PostProcessPipeline
+        let bind_group = render_context
+            .render_device()
+            .create_bind_group(&BindGroupDescriptor {
+                label: Some("post_process_bind_group"),
+                layout: &post_process_pipeline.layout,
+                entries: &[
+                    BindGroupEntry {
+                        binding: 0,
+                        // Make sure to use the source view
+                        resource: BindingResource::TextureView(post_process.source),
+                    },
+                    BindGroupEntry {
+                        binding: 1,
+                        // Use the sampler created for the pipeline
+                        resource: BindingResource::Sampler(&post_process_pipeline.sampler),
+                    },
+                    BindGroupEntry {
+                        binding: 2,
+                        // Set the settings binding
+                        resource: settings_binding.clone(),
+                    },
+                ],
+            });
+
+        // Begin the render pass
+        let mut render_pass = render_context.begin_tracked_render_pass(RenderPassDescriptor {
+            label: Some("post_process_pass"),
+            color_attachments: &[Some(RenderPassColorAttachment {
+                // We need to specify the post process destination view here to make sure we write to the appropriate texture.
+                view: post_process.destination,
+                resolve_target: None,
+                ops: Operations::default(),
+            })],
+            depth_stencil_attachment: None,
+        });
+
+        // Set the render pipeline for this render pass
```

Comment:
We can condense this, it's just wgpu knowledge. Not sure we should explain it, I feel like we should focus on bevy only things.

Reply:
Yeah, it was mostly because it was the only lines that didn't have any comments and it felt a bit weird lol. But I agree that it's not necessary